advisor_alert.rst

```diff
@@ -0,0 +1,29 @@
+Advisor Alerts
+==============
+The Advisor is an entity in Scylla Monitoring that aims to find potential users' problems and advise on how to solve them.
+Currently, it uses low-priority alerts as a notification method.
```

Comment:
First, I would remove "Currently", its not helpful
Second, I would reword to something like:
You can attach Advisor alerts to notification as you do with any other alert (link).
All Advisor alerts are marked as low-priority, and you can change that in file (link)

Reply:
ok about the currently, but this is file is not aim for users, but for developers so they can suggest more advisor alerts